lib/grpc.js

```diff
@@ -148,7 +148,8 @@ GrpcClient.prototype.constructSettings = function constructSettings(
  * @param {number} port - The port of the service.
```

Comment:
It looks like `servicePath` and `port` will be passed into this method twice (explicitly as parameters and implicitly as options). Is there a good way to minimize this repetition?